demo/index.html

```diff
@@ -15,30 +15,28 @@
 
   <title>paper-toolbar demo</title>
 
-  <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
-
-  <script type="module" src="../../iron-demo-helpers/demo-pages-shared-styles.js"></script>
-  <script type="module" src="../../iron-demo-helpers/demo-snippet.js"></script>
-  <script type="module" src="../../iron-flex-layout/iron-flex-layout.js"></script>
-  <script type="module" src="../../iron-icons/iron-icons.js"></script>
-  <script type="module" src="../../paper-icon-button/paper-icon-button.js"></script>
-  <script type="module" src="../../paper-progress/paper-progress.js"></script>
-  <script type="module" src="../../paper-styles/default-theme.js"></script>
-  <script type="module" src="../paper-toolbar.js"></script>
-
-  <!-- FIXME(polymer-modulizer):
-        These imperative modules that innerHTML your HTML are
-        a hacky way to be sure that any mixins in included style
-        modules are ready before any elements that reference them are
-        instantiated, otherwise the CSS @apply mixin polyfill won't be
-        able to expand the underlying CSS custom properties.
-        See: https://github.com/Polymer/polymer-modulizer/issues/154
-        -->
-    <script type="module">
-const $_documentContainer = document.createElement('template');
-
-$_documentContainer.innerHTML = `<custom-style>
-    <style is="custom-style" include="demo-pages-shared-styles">
+  <script src="https://unpkg.com/@webcomponents/webcomponentsjs@^2.0.0/webcomponents-loader.js"></script>
+
+  <script type="module">
+    import '@polymer/iron-demo-helpers/demo-pages-shared-styles.js';
+    import '@polymer/iron-demo-helpers/demo-snippet.js';
+    import '@polymer/iron-flex-layout/iron-flex-layout.js';
+    import '@polymer/iron-icons/iron-icons.js';
+    import '@polymer/paper-icon-button/paper-icon-button.js';
+    import '@polymer/paper-progress/paper-progress.js';
+    import '@polymer/paper-styles/default-theme.js';
+    import '../paper-toolbar.js';
+
+    const $_documentContainer = document.createElement('template');
+
+    $_documentContainer.innerHTML =
+        `<custom-style><style is="custom-style" include="demo-pages-shared-styles"></style></custom-style>`;
```

Comment:
Did you try putting this in the DOM?

Reply:
Yes - I added a suggestion to the migration doc on how to handle `<style is="custom-style" include="demo-pages-shared-styles">`.